metis/api/adaptiveipam/v1/adaptiveipam.proto

```
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package adaptiveipam.v1;
```

Buf (1.66.1)

[error] Files with package "adaptiveipam.v1" must be within a directory "adaptiveipam/v1" relative to root but were in directory "metis/api/adaptiveipam/v1".

(PACKAGE_DIRECTORY_MATCH)